dds/DCPS/Serializer.h

```diff
@@ -69,7 +69,8 @@ String endianness_to_string(Endianness endianness);
 enum Extensibility {
   FINAL,
   APPENDABLE,
-  MUTABLE
+  MUTABLE,
+  ANY_EXTENSIBILITY
```

Comment:
For naming, something like `EXTENSIBILITY_ANY` seems preferred, but it's not a big deal.